### PR TITLE
Incredibly sketchy code that need to be heavily reviewed...but then again it also works very well

### PR DESCRIPTION
**Pull request by SamuelStone-beep**

### CODE REVIEW

File: src/main/java/frc/robot/commands/Autonomous/DriveDistance.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.subsystems.Intake;
+
+public class DriveDistance extends CommandBase {
+  private Drivetrain drivetrain;
+  private double distance;
+  private double leftStart;
+  private double rightStart;
+  private double scale = 1.0;
```
Comment (borkae) [inline src/main/java/frc/robot/commands/Autonomous/DriveDistance.java:19]: Note for later: Move this out to constants for Properties

File: src/main/java/frc/robot/commands/PrintCommand.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    this.message = message;
+    // Use addRequirements() here to declare subsystem dependencies.
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    SmartDashboard.putString("Test Slot", message);
+  }
+}
```
Comment (borkae) [inline src/main/java/frc/robot/commands/PrintCommand.java:28]: Interesting.  PrintCommand is also a library class.  If we want to keep this, lets rename ours.

File: src/main/java/frc/robot/subsystems/Drivetrain.java [hunk 3]
```diff
@@ -91,6 +88,15 @@ public void initializeMotorControllers(){
     leftMaster.getEncoder().setPosition(0);
     rightMaster.getEncoder().setPosition(0);
     
+    leftMaster.setOpenLoopRampRate(0.5);
+    rightMaster.setOpenLoopRampRate(0.5);
+
+    leftMaster.setSmartCurrentLimit(30);
```
Comment (borkae) [inline src/main/java/frc/robot/subsystems/Drivetrain.java:94]: How close is this to allowing the breaker to pop?  Do we need to back off on the right side limits?

**Review by borkae — APPROVED** [review]

File: src/main/java/frc/robot/commands/PrintCommand.java (hunk 2 above)
Comment (SamuelStone-beep) [reply src/main/java/frc/robot/commands/PrintCommand.java:28]: yeah we don't need to keep this, however it was an incredibly helpful tool to test both shuffleboard and button declaration

**Review by duffym22 — APPROVED** [review]

**Pull request merged**